### PR TITLE
write_to_dot updated. Successfully generated tex file from dot file.

### PR DESCRIPTION
**Pull request by Chinmay4400**
Updated plasma.write_to_dot method to generate parsable dot file.
Earlier generating tex from this dot file caused the following errors:
```Traceback (most recent call last):
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 332, in parse_drawstring
    didx, cmd = doeE(c, s[idx + 1:])
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 262, in doeE
    points = [float(t) for t in tokens]
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 262, in <listcomp>
    points = [float(t) for t in tokens]
ValueError: could not convert string to float: 'F'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 3075, in main
    s = conv.convert(dotdata)
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 829, in convert
    self.do_nodes()
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 649, in do_nodes
    s += self.do_drawstring(drawstring, node)
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 536, in do_drawstring
    drawoperations, stat = parse_drawstring(drawstring)
  File "/home/chinmay/anaconda3/envs/tardis/lib/python3.6/site-packages/dot2tex/dot2tex.py", line 347, in parse_drawstring
    log.debug("Failed to parse drawstring %s\n%s", s, err.message)
AttributeError: 'ValueError' object has no attribute 'message'
```
The reason for this error was most likely a newline character being added before the labels (latex formulae) in the plasma graph. Removing the '\n' character successfully generates the tex file.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How is this fix tested?
plasma.write_to_tex function has successfully been used in quickstart.ipynb to generate .tex file.

### CODE REVIEW

**Comment by wkerzendorf** [comment]
okay great - essentially all PRs - should have tests and documentation. I think that is a good next step.

**Comment by wkerzendorf** [comment]
We are missing tests - and documentation. The documentation might be good at https://tardis-sn.github.io/tardis/running/index.html

**Comment by Chinmay4400** [comment]
@wkerzendorf Can you elaborate about the tests...the changes I made are fairly trivial, I'm not sure how to write tests for them. Meanwhile, I'll start working on the documentation.

**Comment by wkerzendorf** [comment]
well it didn't work before and now it works. So there should be a test that reflects it - have a look at pytest. Ultimately, the best practice is to do test-driven development.

File: tardis/plasma/tests/test_write_to_dot.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import tardis
+from tardis import run_tardis
+from tardis.io.atom_data.util import download_atom_data
+import os
+
+import tempfile
+
+#DATA_PATH = os.path.join(tardis.__path__[0], 'plasma', 'tests', 'data')
+
+def run_simulation(yml_file):
```
Comment (wkerzendorf) [inline tardis/plasma/tests/test_write_to_dot.py]: this is already done somewhere else in the tests - have a look at pytest and fixtures

**Comment by wkerzendorf** [comment]
why is the quickstart nb part of this PR?

**Comment by Chinmay4400** [comment]
it was there earlier demonstrating the use of write_to_dot before writing tests. I'll remove the changes to it.

**Comment by Chinmay4400** [comment]
@wkerzendorf I have made the required changes...please let me know how to proceed further.

**Pull request closed without merging**